### DISCUSSION TITLE
Self-update

### DISCUSSION THREAD

**Discussion by sudokillnine** [ImranR98/Obtainium#1709] [Q&A] [answered]
Hello !

A think I am not sure about reading the doc is the self update of Obtainium. 

I have installed it directly from the APK and I don't know if:
- it will update itself automatically
- or if I have to add [its repo](https://github.com/ImranR98/Obtainium) to its app list to get it done

Thank you :)

**Comment by Mikaela** [comment]
Yes, when you install Obtainium it should list itself by default and notify you on updates.

If you are on Android 12 or newer, it will automatically update if it's the latest app to install itself (so you will see at least one prompt on whether to install update).

**Reply by DwainZwerg** [reply, marked as answer]
As a small addition: If you have downloaded the F-Droid version (it is also offered [in the GitHub releases](https://github.com/ImranR98/Obtainium/releases/tag/v1.1.13)), Obtainium is not entered in the list, cannot be entered in the list and does not update itself (because F-Droid is supposed to do this).
If we have answered your question: Could you please close this discussion as resolved? Thank you very much.

**Comment by sudokillnine** [comment]
Okay, thank you for the answer! 

I firstly install it with f-droid, but uninstall it and reinstall it from github.
I still don't have it on the app list, maybe some config data was not erased ?
So at least I know it should be listing itself, I will try to achieve that.

**Reply by DwainZwerg** [reply]
> I still don't have it on the app list, maybe some config data was not erased ?

I rather guess that (as I said) you downloaded the F-Droid version, which is **also offered in the GitHub releases** (the F-Droid version is labeled app-arch-**fdroid**-release.apk and the GitHub version is labeled app-arch-release.apk).

**Reply by sudokillnine** [reply]
Thanks! It is exactly that!
